.github/workflows/test.yaml

```diff
@@ -114,3 +114,37 @@ jobs:
         if: ${{ failure() }}
         run: |
           cat galaxy-${{ matrix.galaxy_version }}/*.log
+  test-toolshed:
```

Comment:
I think we can simplify the CI setup by dropping all the changes to this file, and instead add:
```
          if [ "${{ matrix.galaxy_version }}" = 'dev' ]; then
              export BIOBLEND_TOOLSHED_URL=https://testtoolshed.g2.bx.psu.edu/
          fi
```

to the `Run tests` step above. In this way the TS tests would run just once per Python version, but together with the other Galaxy tests.